int32    robot_ID
char     robot_mode
int32    which_task
int32    which_target
Pose     expect_pos
bool     isvalid

int32    robot_power
float32  move_distance
bool     isupdate
